stable/kommander/values.yaml

```diff
@@ -5,8 +5,8 @@ image:
 replicas: 1
 logoutRedirectPath: /ops/landing
 clusterPollingInterval: 3000
-clientPollingInterval: 3000
 hostPollingInterval: 3000
+clusterPollingThrottleFactor: 1
 kubernetesVersionsSelection: '[{"kubernetesVersion":"1.15.4","konvoyVersion":"v1.3.0-kommander-beta2","kubeaddonsVersion":"kommander-beta2"},{"kubernetesVersion":"1.15.3","konvoyVersion":"v1.1.5","kubeaddonsVersion":"stable-1.15.3-4"}]'
```

Comment:
can you `v1.3.0-kommander-beta2` -> `v1.3.0-kommander-beta3` and `kommander-beta2` -> `kommander-beta3` while you are editing here?  Also, lets wait for the catalog changes to be merged and use that image tag.

Reply:
Yes